docs/contributing/code/building.md

```diff
@@ -67,6 +67,19 @@ Since v1.1.0, Kyuubi support building with different Spark profiles,
 | -Pspark-3.2 | No      | 1.4.0 |
 | -Pspark-3.3 | Yes     | 1.6.0 |
 
+## Building Kyuubi Against Different Scala Versions
+
+Since v1.8.0, Kyuubi support building with different Scala profile. Currently, Kyuubi supports building with Scala 2.12 and 2.13, while Scala 2.12 by default.
+
+|   Profile    | Default | Since |
+|--------------|---------|-------|
+| (Scala 2.12) | ✓       | -     |
```

Comment:
Keep the same style as above?

![image](https://github.com/apache/kyuubi/assets/17894939/d3893d87-8b60-4d40-b5f8-d5b2e3050e2d)